jdaviz/configs/mosviz/tests/test_helper.py

```diff
@@ -208,6 +208,18 @@ def test_load_image(mosviz_app, image):
     assert list(data.keys())[0] == f"{label} 0"
 
 
+def load_single_image_multi_spec(mosviz_app, image, spectrum1d, spectrum2d):
+    spectra1d = [spectrum1d for _ in range(3)]
+    spectra2d = [spectrum2d for _ in range(3)]
```

Comment:
Simpler like this?
```suggestion
    spectra1d = [spectrum1d] * 3
    spectra2d = [spectrum2d] * 3
```

Reply:
Yes, but I was copying from an existing test. Maybe I'll go ahead and change all of them...